File: api/src/main/java/com/ruby/rubia_server/core/dto/DepartmentDTO.java
package com.ruby.rubia_server.core.dto;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

import java.time.LocalDateTime;
import java.util.UUID;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class DepartmentDTO {
    
    private UUID id;
    private UUID companyId;
    private String name;
    private String description;
    private Boolean autoAssign;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
}

File: api-dotnet/Rubia.Server/DTOs/DepartmentDto.cs
namespace Rubia.Server.DTOs;

public class DepartmentDto
{
    public Guid Id { get; set; }
    public Guid CompanyId { get; set; }
    public string Name { get; set; } = string.Empty;
    public string? Description { get; set; }
    public bool AutoAssign { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}